tasks/create-advisory-internal-request/create-advisory-internal-request.yaml

```diff
@@ -92,3 +96,5 @@ spec:
           echo "$results"
           exit 1
         fi
+
+        echo -n "Advisory URL: $(echo ${results} | jq -r '.advisory_url')" | tee $(results.advisory_url.path)
```

Comment:
Are we confident there will always be a url (as long as the result is `"Success"`)? If not, we would end up with `null` as the resulting string. Maybe at least use `""` in such case? Using `jq -r '.advisory_url // ""'`.

Reply:
The result has to exist or the IR pipelinerun would've failed. It will be the empty string or the url. What do I get from adding in the empty string default if the string is already that if not the url?

Reply:
I guess I meant that what if the wrong version of the internal cluster task is used that doesn't have it yet. But I guess that's unlikely.

Reply:
I got it, maybe the IR blows up completely. I will update